include/dslash.h

```diff
@@ -433,7 +436,7 @@ namespace quda
       auto aux_ = (arg.pack_blocks > 0 && (arg.kernel_type == INTERIOR_KERNEL || arg.kernel_type == UBER_KERNEL)) ?
         aux_pack :
         ((arg.shmem > 0 && arg.kernel_type == EXTERIOR_KERNEL_ALL) ? aux_barrier : aux[arg.kernel_type]);
-      return TuneKey(in.VolString().c_str(), typeid(*this).name(), aux_);
+      return TuneKey(in[0].VolString().c_str(), typeid(*this).name(), aux_);
```

Comment:
Could add `VolString()` to `vector_ref`

Reply:
a805ab08ae34c093a76f3b68efd636bafd4403cc